tests/test_format.py

```diff
@@ -225,26 +225,27 @@ def test_format_comparison_bars_to_js_exceptions(self):
             self.comparison_coords_headers, self.comparison_coords_data, 11)
 
     def test_format_emperor_html_footer_string(self):
+        self.maxDiff=5000
         """Test correct formatting of the footer string"""
         # footer for a jackknifed pcoa plot without biplots
         out_string = format_emperor_html_footer_string(False, True)
-        self.assertEqual(out_string, EXPECTED_FOOTER_A)
+        self.assertItemsEqual(out_string.split('/'), EXPECTED_FOOTER_A.split('/'))
```

Comment:
I see, this makes sense! A follow-up question, why did you choose '/'?
'\n' makes more sense to me, but then again, I haven't yet used this in
a real-case scenario.

On (Mar-04-15|15:37), mortonjt wrote:
> >          """Test correct formatting of the footer string"""
> >          # footer for a jackknifed pcoa plot without biplots
> >          out_string = format_emperor_html_footer_string(False, True)
> > -        self.assertEqual(out_string, EXPECTED_FOOTER_A)
> > +        self.assertItemsEqual(out_string.split('/'), EXPECTED_FOOTER_A.split('/'))
> 
> Its just a way to make debugging easier.  Before, an entire HTML page would be dumped on the page.  Using this will allow only portions of differing HTML to be displayed.
> 
> It isn't a big though - I can remove it.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/352/files#r25826331